DDDEastAnglia.Tests/SponsorBuilder.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using DDDEastAnglia.DataAccess.SimpleData.Models;
+
+namespace DDDEastAnglia.Tests
+{
+    internal class SponsorBuilder
+    {
+        private byte[] logo;
+        private string name;
+        private DateTime? paymentDate;
+        private bool showPublicly = true;
+        private int sponsorId;
+        private int sponsorshipAmount;
+        private string url;
+
+        public Sponsor Build()
```

Comment:
I tend to put this at the bottom, as it's the least interesting bit of a Builder (to me). 

I'm also not convinced a Builder brings much valuable to a publicly-mutable type like `Sponsor`, unless it's doing a bit more than simply setting the properties on the type. (Mark Seemann terms Builder methods like these "higher-order Builder methods"; I'm not sure that's technically correct, but let's roll with it.)

Higher-order Builder methods for Sponsor might be something like `AtPlatinumLevel()` - something more related to our domain than just what they've paid and when.